storage/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3.7"
 services:
 
   spdk:
-    image: ghcr.io/opiproject/opi-spdk:main
```

Comment:
this is doing `latest` why not specify a correct tag ?

Reply:
I wanted to prove this is why the double build was happening. I think we need to pass it in as an argument here so we can test with the tagged version we just built in the CI.